.github/workflows/tb_plugin_ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       matrix:
         python-version: [3.6, 3.7, 3.8, 3.9]
         cuda-version: [cpu, cu101, cu102, cu111]
-        pytorch-version: [nightly, 1.8rc]
+        pytorch-version: [nightly, stable]
```

Comment:
we will need to support multiple PT versions in the near future. e.g. test against 1.8, 1.9 to check compatibility
only a `stable` seems not enough

Reply:
Yes, I modify it to [nightly, rc1.8, stable], and the nightly is 1.9, stable is 1.8.